src/configurations/destinations/xpixel/db-config.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "XPIXEL",
+  "displayName": "XPixel",
+  "config": {
+    "transformAtV1": "processor",
+    "saveDestinationResponse": true,
+    "includeKeys": [
+      "pixelId",
+      "eventsMapping",
+      "blacklistedEvents",
+      "whitelistedEvents",
+      "eventFilteringOption",
+      "oneTrustCookieCategories"
+    ],
+    "excludeKeys": [],
+    "supportedSourceTypes": ["web"],
+    "supportedConnectionModes": {
+      "web": ["device"]
+    },
+    "supportedMessageTypes": {
+      "device": {
+        "web": ["track", "page"]
+      }
+    },
+    "destConfig": {
+      "defaultConfig": [
+        "pixelId",
+        "eventsMapping",
+        "blacklistedEvents",
+        "whitelistedEvents",
+        "eventFilteringOption",
+        "oneTrustCookieCategories"
+      ],
+      "web": ["useNativeSDK", "connectionMode"]
+    },
+    "secretKeys": ["pixelId"]
```

Comment:
@aanshi07 Is PixelId sensitive key being used for auth ?

Reply:
@krishna2020 For admin dashboard we can say that but from network tab anyone can see it.